applications/pelion_client/src/modules/Kconfig.power_manager

```diff
@@ -37,6 +38,19 @@ config PELION_CLIENT_POWER_MANAGER_ERROR_TIMEOUT
 	help
 	 Time in seconds after which the device will be turned off after an error.
 
+config PELION_CLIENT_POWER_MANAGER_MODULES_COUNT
```

Comment:
This is not used anywhere, it's only asserted :)  `module_flags` uses `CONFIG_CAF_MODULES_FLAGS_COUNT` instead

Reply:
Removed